### ISSUE TITLE
GCS v 4.16.0 fails to launch on Windows 10

### ISSUE THREAD

**Issue by TomSmeagol** [richardwilkes/gcs#192]
After the newest update, GCS will not launch on Windows 10. I receive no error messages, and GCS does not appear in Task Manager. It just simply does nothing.

**Comment by richardwilkes** [comment]
This is the second report I've had of not being able to launch on Windows 10. I am using a more recent version of Java, which may need the latest Windows updates to work properly. Please try updating Windows and trying again to see if that helps. Either way, I'd like to know what the result was.

**Comment by TomSmeagol** [comment]
Unfortunately, I have no Windows Updates to install---my system is already up to date. GCS 4.16.0 still does not launch. And just to make sure I hadn't done something screwy with my configuration, I uninstalled 4.16.0, downloaded 4.15.1, and installed it. It DOES launch normally.

**Comment by richardwilkes** [comment]
OK. The only real difference for purposes of running is the use of Java 15 rather than 14. I upgraded to this (unreleased) version to fix a number of outstanding bugs specifically in the Windows version of the packager. My system has no problems with it, so I assumed it worked in general. Odd that yours (and one other at this point) doesn't work, with no real explanation (the other person gave me a log file that just showed the program exits pretty much immediately upon launch due to trying to execute through a bad region of memory).

**Comment by richardwilkes** [comment]
I'll have to try switching back to Java 14 and just accepting the bugs that it has as things I can't get around. Not ideal, but better than not working at all for some percentage of folks. Wish there was a way to pinpoint what the actual issue is -- it runs on a fair number of Windows 10 installations with no issue, so there is something specific about yours and this other person's setup that is causing the problem.

**Comment by Belatucadras** [comment]
[hs_err_pid11236.log](https://github.com/richardwilkes/gcs/files/4661027/hs_err_pid11236.log)

It also will not work for me using Windows 10. The Text Document above is the error I get.

**Comment by richardwilkes** [comment]
I’ve had a handful of other reports of failure like yours now. I plan to put together an alternative version of the release that swaps the Java version back to 14. That will unfortunately cause a few fixes that were made in this release to address issues on Windows to break again, but that’s better than not being able to run at all.

**Comment by CaydenT** [comment]
Same thing is happening to me in Windows 8.1 just downloaded GCS newest version and launching it does..... nothing. No error, no nothing.

**Comment by Ellywelly** [comment]
Yup, much the same error on Windows 7. Can't launch.

Can I just say how amazing it is this is still getting major updates to this day, though?

**Comment by marcellraba** [comment]
Win10 Pro x64 version, latest experimental build, CTD on start.

**Comment by Ghostwalker7** [comment]
Same thing here. I updated Windows (10) and Java, but the program still doesn't start. I tried to uninstall and install again, restart the PC, but the results are the same.

**Comment by TomSmeagol** [comment]
FWIW I also tried the new version on my laptop, which has Windows 10 but of course a very different configuration otherwise with the same result as my desktop computer. Immediate close to desktop.

--Tom

On May 21, 2020, at 9:40 AM, Ghostwalker7 <notifications@github.com> wrote:

﻿

Same thing here. I updated Windows (10) and Java, but the program still doesn't start. I tried to uninstall and install again, restart the PC, but the results are the same.
